packages/devtools_app/lib/src/screens/memory/panes/control/source_dropdown.dart

```diff
@@ -67,10 +65,6 @@ class _MemorySourceDropdownState extends State<MemorySourceDropdown>
       value: controller.memorySource,
       onChanged: (String? newValue) {
         setState(() {
-          ga.select(
-            analytics_constants.memory,
-            analytics_constants.sourcesDropDown,
-          );
```

Comment:
are we removing this because we are removing the SourcesDropdown feature?

Reply:
Yes, as we agreed in https://github.com/flutter/devtools/issues/4645, we will redesign the feature. I do not see value in maintaining analytics for it.